app/src/main/java/app/revanced/integrations/patches/spoof/requests/StoryBoardRendererRoutes.java

```diff
@@ -1,22 +1,21 @@
 package app.revanced.integrations.patches.spoof.requests;
 
-import app.revanced.integrations.requests.Requester;
-import app.revanced.integrations.requests.Route;
-
 import java.io.IOException;
 import java.net.HttpURLConnection;
 
-/** @noinspection unused*/
-public final class StoryBoardRendererRoutes {
+import app.revanced.integrations.requests.Requester;
+import app.revanced.integrations.requests.Route;
+
+final class StoryBoardRendererRoutes {
     private static final String YT_API_URL = "https://www.youtube.com/youtubei/v1/";
     private static final String YT_API_KEY = "AIzaSyA8eiZmM1FaDVjRy-df2KTyQ_vz_yYM39w";
 
-    static final Route GET_PLAYER_RESPONSE_BODY = new Route(Route.Method.POST, "player?key={api_key}");
+    private static final Route GET_PLAYER_RESPONSE_BODY = new Route(Route.Method.POST, "player?key={api_key}");
 
     private StoryBoardRendererRoutes() {
     }
 
-    public static HttpURLConnection getPlayerResponseConnectionFromRoute() throws IOException {
+    static HttpURLConnection getPlayerResponseConnectionFromRoute() throws IOException {
```

Comment:
Following the same convention as the other route classes, this method should have one `Route` parameter. Then the route from above (should be made public) can be passed to it.